### ISSUE TITLE
Flower_plot connection column should be the same as tags in the PPT

### ISSUE THREAD

**Issue by j-harbin** [dfo-mar-mpas/MarConsNetAnalysis#6]
Right now the Flower_plot connection are as follows

![image](https://github.com/user-attachments/assets/9046daa6-0cbc-49ee-95bc-4cf734fbefaa)

The tags in the ppt differ slightly. E.g. for Environmental & Representativity the tag in the PPT is the following: "Environmental (Representativity)".

It is still possible to link the indicator bins from the csv to the projects in the PPT, but it seems more intuitive if they were the same. @remi-daigle , I tagged you to do this because I didn't want to change anything in your sandbox.

PS  a new om and salary have been pulled with `keep=TRUE` from the PPT yesterday so you should be able to just load up the rds saved in the IN folder to see the new tags.

**Comment by j-harbin** [comment]
Through further inspection, it may just be the above example that is the problem. See below (ps the ones that do not have a match just mean there is no projects assigned to them.

![image](https://github.com/user-attachments/assets/e47cd7bc-7cce-44fe-b2e7-5294801d5d59)

**Comment by j-harbin** [comment]
@remi-daigle , I have my own version of this now so I assigned myself instead :)

**Comment by remi-daigle** [comment]
Yeah, I think it's because we haven't settled on a name for the Environmental one. On the whiteboard, we started with JUST the word Environmental, but then we realized that the bin was about Representativity. It initially became Environmental/Representativity, and I think I wrote down Environmental & Representativity in my first flower plot and Tana went with Environmental (Representativity) in the PPT. All that being said, I think it should just be "Environmental Representativity" everywhere, but I'm not sure what is correct anymore!
